Although most libfprint tests are vert quick, several of the VirtualImage
ones come very close to the default time-out threshold of 30 seconds.
Increase the relevant threshold to 45 seconds to make those tests less
sensitive to CPU frequency, system load etc.

Index: libfprint-v1.94.6/tests/meson.build
===================================================================
--- libfprint-v1.94.6.orig/tests/meson.build
+++ libfprint-v1.94.6/tests/meson.build
@@ -110,6 +110,7 @@ if get_option('introspection')
                     suite: ut_suite,
                     depends: libfprint_typelib,
                     env: envs,
+                    timeout: 45,
                 )
             endforeach
 
